contrib/storage-hive/hive-exec-shade/pom.xml

```diff
@@ -120,6 +120,10 @@
           <groupId>org.codehaus.jackson</groupId>
           <artifactId>jackson-xc</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>io.airlift</groupId>
+          <artifactId>aircompressor</artifactId>
+        </exclusion>
```

Comment:
Is there any reason for excluding `aircompressor` here if you have relocated it below?